web/src/stats/stats.js

```diff
@@ -872,6 +880,8 @@ function populate_number_of_users(data) {
 }
 
 function populate_messages_read_over_time(data) {
+    // Content rendered by this method is titled as "Messages read over time" on the webpage
+    
```

Comment:
There are some linter errors for this commit.

```diff
diff --git a/web/src/stats/stats.js b/web/src/stats/stats.js
index bf5fd3ef1f..6ea2325f2e 100644
--- a/web/src/stats/stats.js
+++ b/web/src/stats/stats.js
@@ -210,7 +210,7 @@ function set_guest_users_statistic(guest_users) {
 // PLOTLY CHARTS
 function populate_messages_sent_over_time(data) {
     // Content rendered by this method is titled as "Messages sent over time" on the webpage
-    
+
     if (data.end_times.length === 0) {
         // TODO: do something nicer here
         return;
@@ -519,7 +519,7 @@ function compute_summary_chart_data(time_series_data, num_steps, labels_) {
 
 function populate_messages_sent_by_client(data) {
     // Content rendered by this method is titled as "Messages sent by client" on the webpage
-    
+
     const layout = {
         width: 750,
         height: null, // set in draw_plot()
@@ -661,7 +661,7 @@ function populate_messages_sent_by_client(data) {
 
 function populate_messages_sent_by_message_type(data) {
     // Content rendered by this method is titled as "Messages sent by recipient type" on the webpage
-    
+
     const layout = {
         margin: {l: 90, r: 0, b: 10, t: 0},
         width: 750,
@@ -881,7 +881,7 @@ function populate_number_of_users(data) {
 
 function populate_messages_read_over_time(data) {
     // Content rendered by this method is titled as "Messages read over time" on the webpage
-    
+
     if (data.end_times.length === 0) {
         // TODO: do something nicer here
         return;
```

Reply:
that's nuanced. Fixed, thanks!